omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -290,6 +290,18 @@
     end
   end
 
+  block "Remove .pyi files" do
+    # We don't need them at runtime
```

Comment:
Nit 🤏 for folks who may not necessarily know:
```suggestion
    # These are files containing Python type annotations which aren't used at runtime
```